client/components/back-button/index.d.ts

```diff
@@ -0,0 +1,10 @@
+import { FC, MouseEvent } from 'react';
+
+declare interface BackButtonProps {
+	onClick?: ( event: MouseEvent< HTMLButtonElement > ) => void;
```

Comment:
Some of those might be easier to just convert to TS instead of adding module declarations. `BackButton` is a pretty straightforward one. Have you considered that?

Reply:
I considered it but honestly I only wanted to focus on the quick win. This PR will land on another branch and I'd wouldn't like to add more code to it. 

Anyway, I can do it if that's a proper and better solution.